packages/webapp/pages/squads/[handle]/[token].tsx

```diff
@@ -228,5 +239,5 @@ export function getStaticProps({
 }: GetStaticPropsContext<
   SquadReferralProps & ParsedUrlQuery
 >): GetStaticPropsResult<SquadReferralProps> {
-  return { props: { ...params } };
+  return { props: { ...params }, revalidate: 60 };
```

Comment:
What I originally meant is to have the query right here for all the information.
Many crawlers don't support javascript meaning any change we apply to OG during client rendering will not take affect.
Thanks to ISR, we can do it server-side and cache it.